<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JSONedit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Text</string>
       </attribute>
       <widget class="CodeEditor" name="textEdit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>851</width>
          <height>441</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tree</string>
       </attribute>
       <widget class="TreeWidget" name="treeWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>671</width>
          <height>451</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>10</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Nodetype</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>40</y>
          <width>181</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>null</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>integer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>float point</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>string</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>boolean</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>array</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>object</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>100</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_name">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>130</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>200</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>value</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_value">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>230</y>
          <width>181</width>
          <height>87</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>Search</string>
    </property>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSearch"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionNew">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>压缩</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>还原</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">treewidget.h</header>
  </customwidget>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
